be/src/common/config.cpp

```diff
@@ -60,6 +60,9 @@ DEFINE_Int32(be_port, "9060");
 // port for brpc
 DEFINE_Int32(brpc_port, "8060");
 
+// port for brpc
+DEFINE_Int32(arrow_flight_port, "8070");
```

Comment:
add the config to conf/be.conf then user will know there is a port he should config.